modules/gapi/include/opencv2/gapi/render/render.hpp

```diff
@@ -97,6 +97,17 @@ void GAPI_EXPORTS render(cv::Mat& y_plane,
                          const Prims& prims,
                          cv::GCompileArgs&& args = {});
 
+/** @brief The function renders on the input media frame passed drawing primitivies
+
+@param NV12 input Media Frame :  @ref cv::MediaFrame.
+@param prims vector of drawing primitivies
+@param args graph compile time parameters
+*/
+void GAPI_EXPORTS render(cv::MediaFrame& NV12,
```

Comment:
NV12 parameter name is misleading. With such signature and function name MediaFrame here can be of the different (non nv12) format, so this function should handle all possible formats (maybe with an assert inside on which format support is currently implemented), or have a different name

Reply:
Done